.gitignore

```diff
@@ -5,6 +5,7 @@ tools/crd-generator/crd-generator
 tools/csv-generator/csv-generator
 tools/resource-generator/resource-generator
 tools/marketplace/marketplace
+tools/gosec/gosec
```

Comment:
This creates the impression that we are having our own gosec tool here, we should rename it to make clear what it does.